connector-runtime/spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/OutboundConnectorsAutoConfiguration.java

```diff
@@ -39,6 +45,9 @@
 @EnableConfigurationProperties(ConnectorProperties.class)
 public class OutboundConnectorsAutoConfiguration {
 
+  @Value("${camunda.connector.secret-provider.discovery.enabled:true}")
+  Boolean secretProviderLookupEnabled;
```

Comment:
I would add this one and `camunda.connector.secret-provider.environment.enabled` to [ConnectorProperties](https://github.com/camunda/connectors/blob/68a47f72e82c3b284ba838c6ca7da46bf44038cb/connector-runtime/spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/ConnectorProperties.java) to enable autocompletion in properties files and keep all our properties in the same place.

I think the `additional-spring-configuration-metadata` file won't be needed then right?

Reply:
Thanks for the hint. Fixed!